e2e/test-static-pod.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -o errexit
+
+echo "Creating network attachment definition"
+kubectl create -f static-pod-nad.yml
+
+echo "Creating static pod config file"
+docker cp simple-static-pod.yml multus-whereabouts-worker:/etc/kubernetes/manifests/static-web.yaml
+
+echo "Configuring kubelet on the node to use correct config file"
+docker cp static-pod-kubelet multus-whereabouts-worker:/etc/kubernetes/kubelet
+
+echo "Restarting kubelet"
+docker exec -it multus-whereabouts-worker /bin/bash -c "systemctl restart kubelet"
```

Comment:
To change the kubelet config, modify/restart could be a way, but I suppose we could configure these parameter at cluster creation (and it could be simple). This config does NOT conflict any test scenarios so we can configure it at the beginning of e2e testing. How about to modify 'setup_cluster.sh' to create kind cluster with this config?

Here are the pointers for that:
- https://kind.sigs.k8s.io/docs/user/configuration/#kubeadm-config-patches
- https://kubernetes.io/docs/reference/config-api/kubeadm-config.v1beta3/
- https://github.com/kubernetes/release/blob/master/cmd/kubepkg/templates/latest/rpm/kubeadm/10-kubeadm.conf#L7-L9